src/decode.h

```diff
@@ -254,9 +254,10 @@ typedef struct {
 
 #define HISTORY_SIZE 16
 typedef struct {
-    uint32_t PC;
-    struct rv_insn *branch_target;
-} branch_history_entry_t;
+    uint8_t idx;
+    uint32_t PC[HISTORY_SIZE];
+    struct rv_insn *branch_target[HISTORY_SIZE];
```

Comment:
Rename it `target` from `branch_target` since it explicitly appears in a structure dealing with branch history."